tests/post/features/pods_alive.feature

```diff
@@ -0,0 +1,5 @@
+@post @ci @local
+Feature: Pods should be alive
+    Scenario: get pods
+        Given The 'kubelet' service is up
+        Then the 'pods' list should not be empty in the 'kube-system' namespace
```

Comment:
This is not really a result of `kubelet` running ;) Need a bit more than that.

Reply:
I dont see a good `given` instead of running the whole bootstrap, thing is these are post install tests so I dont know really, any suggestions ?

Reply:
`Given the Kubernetes API is available`?